src/main/java/org/commcare/cases/instance/CaseInstanceTreeElement.java

```diff
@@ -27,6 +27,10 @@
     private final static XPathPathExpr CASE_INDEX_EXPR = XPathReference.getPathExpr("index/*");
     private final static XPathPathExpr OWNER_ID_EXPR = XPathReference.getPathExpr("@owner_id");
     private final static XPathPathExpr EXTERNAL_ID_EXPR = XPathReference.getPathExpr("@external_id");
+    private final static XPathPathExpr CATEGORY_EXPR = XPathReference.getPathExpr("category");
+    private final static XPathPathExpr STATE_EXPR = XPathReference.getPathExpr("state");
+    private final static XPathPathExpr PATIENT_TYPE_EXPR = XPathReference.getPathExpr("patient_type");
```

Comment:
I think you don't actually want to add these here for aliasing, see my comment on the Case model below

Reply:
So the idea would be that you have to access the index with `@category` but it will find `patient_type` on the backend, rather than either expression being picked up? I did it up here because that is how `CASE_ID_EXPR_TWO` does it and I was basing the aliasing off that. It seems like this is necessary for either expression to be accepted in the app, but if that is not what you had intended, I can remove this.

Reply:
`CASE_ID_EXPR` and `CASE_ID_EXPR_TWO` are semantically identical, so they aren't aliases in the same way that these would be. The new "alias" approach would be like what you described. The `@category` attribute would be a virtual property that is present if either `/category` or `/patient_type` are defined, and contains the value of `/category` if it exists, and `/patient_type` if it doesn't.